### ISSUE TITLE
Multiple file selection

### ISSUE THREAD

**Issue by Tomgroot** [trippo/ResponsiveFilemanager#206]
Can multiple file selection be added? With some sort of selection mode, where you can check files and then click on submit to send it to the input. Check the image:
![Uploading Knipsel.PNG…]()

**Comment by frinkz** [comment]
Yes please, would love this feature. it's the only thing missing. Any comment on this @trippo ?

**Comment by keennang** [comment]
Want this so badly. Is this coming any time soon?

**Comment by judsonbomfim** [comment]
expected to launch this feature?

**Comment by WordpressArtist** [comment]
hello is this feature is released , i  am not able to get this feature , how can i get this feature plz help

**Comment by bastienff** [comment]
i also would like to know if this feature is released or if it will be released

**Comment by fahjan** [comment]
I need this feature please

**Comment by krychu90** [comment]
+1

**Comment by thanhthangnet** [comment]
+1, this feature much important

**Comment by volfrf** [comment]
+1

Any information when this feature should be added (I hope it will be)? Thank you :)

**Comment by judsonbomfim** [comment]
This feature has already been added in the latest version (9.11) :)

**Comment by volfrf** [comment]
Hi,
oh, really? That's great. But I can't find how it works in documentation, can you help me please? :)

Thank you.

**Comment by judsonbomfim** [comment]
Excuse me! The multiple selection is for shipping only. I had done a little test, but I see that I was wrong. I was counting on this for my projects :(

**Comment by fahjan** [comment]
how can i buy this feature ? I need it

**Comment by jarodium** [comment]
well? is it available or not?

**Comment by a1iraxa** [comment]
When will it be available?

**Comment by Pyreweb** [comment]
Hi everybody ! This feature would be so nice...
That will even be greater if we can move many files into a folder in one shot.
Is there any of these improvements scheduled ?

**Comment by kostas1986** [comment]
+1
